test/common/db_seed.js

```diff
@@ -21,6 +21,14 @@ let accounts = [];
 let blocks = [];
 
 class DatabaseSeed {
+	static getBlocks() {
+		return blocks;
+	}
+
+	static getAccounts() {
+		return accounts;
+	}
+
 	static seedAccounts(db) {
 		for (let i = 0; i < 5; i++) {
```

Comment:
Magic number again. As per the links I gave earlier, the suggestion is to declare a const with the value, and some comment explaining the meaning, and then use it, but do not use blind numbers directly.

Reply:
@vitaly-t This code is already there from start. Although added comment to this line in a previous PR.